x-pack/test/security_solution_cypress/cypress/e2e/detection_response/rule_creation/custom_query_rule.cy.ts

```diff
@@ -241,109 +244,104 @@ describe('Custom query rules', { tags: [tag.ESS, tag.BROKEN_IN_SERVERLESS] }, ()
   describe('Custom detection rules deletion and edition', () => {
     context('Deletion', () => {
       beforeEach(() => {
-        deleteAlertsAndRules();
```

Comment:
Why are we deleting this method from the beforeEach hook?

Reply:
`deleteAlertsAndRules()` has been added on line `125` inside `beforeEach` hook so we don't need to invoke the same function here twice.

Reply:
Well done :D